crates/loro-internal/src/encoding/encode_reordered.rs

```diff
@@ -2077,11 +2117,12 @@ mod value {
             &mut self,
             keys: &[InternalString],
             cids: &[ContainerID],
+            id: ID,
```

Comment:
We don't need this here actually. It should be invalid to create Container type in rich text mark value. We can assert that value is not a Container when decoding.

Reply:
I know, but `read_value_content` needs `id` as a parameter.
I noticed that the `read()` function is not used, why is this function kept there?

Reply:
It can be removed.